app/src/controllers/chefs.ts

```diff
@@ -6,44 +6,79 @@ import { IdentityProvider } from '../components/constants';
 
 import type { JwtPayload } from 'jsonwebtoken';
 import type { NextFunction, Request, Response } from '../interfaces/IExpress';
-import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionFormExport } from '../types';
+import type { ChefsFormConfig, ChefsFormConfigData, ChefsSubmissionForm, ChefsSubmissionFormExport } from '../types';
 
 const controller = {
   getFormExport: async (req: Request, res: Response, next: NextFunction) => {
     try {
       const cfg = config.get('server.chefs.forms') as ChefsFormConfig;
-      let formData = new Array<ChefsSubmissionFormExport>();
 
-      // Get a JSON export of all form data merged into a single array
-      await Promise.all(
+      let exportData: Array<Partial<ChefsSubmissionForm & { formId: string }>> = await Promise.all(
         Object.values<ChefsFormConfigData>(cfg).map(async (x: ChefsFormConfigData) => {
-          const data = await chefsService.getFormExport(x.id);
-          data.forEach((d: ChefsSubmissionFormExport) => (d.form.id = x.id));
-          formData = formData.concat(data);
+          return (await chefsService.getFormExport(x.id)).map((data: ChefsSubmissionFormExport) => {
```

Comment:
i wonder if a lot of your types and functions could have simpler and clearer names and jsdocs. 
eg: the type `chefsSubmissionFormExport` could be 'chefsSubmission.
splitting the chefs calls into a separate service would also help with this.

Reply:
Thanks for bringing this up. Confirms what I was thinking that some renaming is required. I've contemplated it for a bit. Will look into it in a future PR.